aws_advanced_python_wrapper/utils/properties.py

```diff
@@ -76,390 +78,500 @@ class WrapperProperties:
-        180_000)  # 3 minutes
+        180_000,
+    )  # 3 minutes
     BG_SUSPEND_NEW_BLUE_CONNECTIONS = WrapperProperty(
         "bg_suspend_new_blue_connections",
         "Enables Blue/Green Deployment switchover to suspend new blue connection requests while the "
         "switchover process is in progress.",
-        False)
+        False,
+    )
 
     # Telemetry
     ENABLE_TELEMETRY = WrapperProperty(
-        "enable_telemetry",
-        "Enables telemetry and observability of the driver.",
-        False
+        "enable_telemetry", "Enables telemetry and observability of the driver.", False
     )
 
     TELEMETRY_SUBMIT_TOPLEVEL = WrapperProperty(
         "telemetry_submit_toplevel",
         "Force submitting traces related to Python calls as top level traces.",
-        False
+        False,
     )
 
     TELEMETRY_TRACES_BACKEND = WrapperProperty(
         "telemetry_traces_backend",
         "Method to export telemetry traces of the driver.",
-        None
+        None,
     )
 
     TELEMETRY_METRICS_BACKEND = WrapperProperty(
         "telemetry_metrics_backend",
         "Method to export telemetry metrics of the driver.",
-        None
+        None,
     )
 
     TELEMETRY_FAILOVER_ADDITIONAL_TOP_TRACE = WrapperProperty(
         "telemetry_failover_additional_top_trace",
         "Post an additional top-level trace for failover process.",
-        False
+        False,
     )
 
     READER_INITIAL_HOST_SELECTOR_STRATEGY = WrapperProperty(
         "reader_initial_connection_host_selector_strategy",
         "The strategy that should be used to select a new reader host while opening a new connection.",
-        "random")
+        "random",
+    )
 
     OPEN_CONNECTION_RETRY_TIMEOUT_MS = WrapperProperty(
         "open_connection_retry_timeout_ms",
         "Maximum allowed time for the retries opening a connection.",
-        30000
+        30000,
     )
 
     OPEN_CONNECTION_RETRY_INTERVAL_MS = WrapperProperty(
         "open_connection_retry_interval_ms",
         "Time between each retry of opening a connection.",
-        1000
+        1000,
+    )
+
+    # Simple Read/Write Splitting
+    SRW_READ_ENDPOINT = WrapperProperty(
+        "srw_read_endpoint",
+        "The read-only endpoint that should be used to connect to a reader.",
+        None,
+    )
+
+    SRW_WRITE_ENDPOINT = WrapperProperty(
+        "srw_write_endpoint",
+        "The read-write/cluster endpoint that should be used to connect to the writer.",
+        None,
+    )
+
+    SRW_VERIFY_NEW_CONNECTIONS = WrapperProperty(
+        "srw_verify_new_connections",
+        "Enables role-verification for new connections made by the Simple Read/Write Splitting Plugin..",
+        True,
+    )
+
+    SRW_VERIFY_INITIAL_CONNECTION_TYPE = WrapperProperty(
+        "srw_verify_initial_connection_type",
+        "Force to verify an initial connection to be either a writer or a reader.",
+        None,
+    )
```

Comment:
Can we also mention the valid input here and in the docs? I initially thought this is a boolean value until I saw how this is used
What about
```
The role of the initial connection. Valid values are 'reader' or 'writer'. If this value is set, the wrapper will verify whether the initial connection matches the specified type.
```